internal/handshake/cipher_suite.go

```diff
@@ -12,7 +12,9 @@ import (
 
 // These cipher suite implementations are copied from the standard library crypto/tls package.
 
-const aeadNonceLength = 12
+const (
+	aeadNonceLength = 12
+)
```

Comment:
Perhaps we can revert these changes to reduce chances of merge conflicts in the future.

Reply:
reverted